code/modules/cargo/packs/organic.dm

```diff
@@ -300,8 +300,9 @@
 	cost = CARGO_CRATE_VALUE * 8
 	crate_type = /obj/structure/closet/crate
 	contains = list(/obj/item/stack/sheet/mineral/coal/five,
```

Comment:
would you mind fixing this list to be properly mulitlined while you're adding stuff to this list?